bin/deploy

```diff
@@ -164,12 +165,30 @@ main() {
 
   local conda_packages
   conda_packages="conda${pyver_prefix}_${deploy_mode}-${pkg_postfix}.txt"
-  local env_file="${SCRIPT_DIR}/../etc/${conda_packages}"
-  local lsstsw_ref
-  lsstsw_ref=$(git log -n 1 --pretty=format:'%h' "$env_file")
+  #local env_file="${SCRIPT_DIR}/../etc/${conda_packages}"
+  #local lsstsw_ref
+  #lsstsw_ref=$(git log -n 1 --pretty=format:'%h' "$env_file")
+  #
+  #echo "::: effective LSST_LSSTSW_REF: ${lsstsw_ref}"
+
+  cd "$LSSTSW"
+  # conda environment reference
+  local env_file="${LSSTSW}/env/scipipe_conda_env/etc/${conda_packages}"
 
-  echo "::: effective LSST_LSSTSW_REF: ${lsstsw_ref}"
+  echo "::: conda environment file: ${env_file}"
+
+  local env_ref
+  cd env
+  if [ -d 'scipipe_conda_env' ]; then
+    echo ":::  conda environment already present"
+  else 
+    git clone "$LSST_BUILD_GITREPO"
```

Comment:
Cloning the repo is is problematic as it will not update if/when `deploy` is re-run.  I think it would more reliable to use the same logic as `newinstall.sh` and fetch the env file by ref.

Reply:
The intention is to keep the same logic is implemented now, in order to fulfill the scope of RFC-553. Changing the logic is a different issue.

Reply:
The current usage model, which includes how jenkins is using `lsstsw`, is that the working `lsstsw` clone is pulled and the `./bin/deploy` is re-run to update the science pipelines biuld environment.  As an example, jenkins is doing this as the start of every `stack-os-matrix` build.  Changing this model should involve documenting a new end-user workspace and updating the jenkins job(s) that use `lsstsw`.